internal/hook/wait_exec_hook_handler.go

```diff
@@ -156,10 +161,10 @@ func (e *DefaultWaitExecHookHandler) HandleHooks(
 				)
 				// Check the individual hook's wait timeout is not expired
 				if hook.Hook.WaitTimeout.Duration != 0 && time.Since(waitStart) > hook.Hook.WaitTimeout.Duration {
-					err := fmt.Errorf("hook %s in container %s expired before executing", hook.HookName, hook.Hook.Container)
+					err := fmt.Errorf("hook %s in container %s expired before executing, hook commands: %+v", hook.HookName, hook.Hook.Container, hook.Hook.Command)
```

Comment:
I think adding hook command here in log has no value add. Better message to float would be something like this `hook %s in container %s expired before execution. This occurred when the hook expired because the container did not become ready within the specified wait time.`

Reply:
Adding hook command here is to let users specifically know which hook commands didn't execute correctly.  I think it's useful for pod annotation hooks since user can identify the hook by checking the hook command.

At least, Adding hook command doesn't hurt to current workflow, right?

Reply:
yes makes sense.

Reply:
just have 1 concern around sensitivity of the data being pushed into logs. 
The commands can have secrets for auth etc. Which might not be very wise to put in logs.

Reply:
Ahh, yeah, your concern is valid, let me remove the commands from logs.